pystarport/pystarport/cluster.py

```diff
@@ -698,7 +740,7 @@ def init_cluster(
                 config.keys(), list(rly_section.get("paths", {}).keys())
             ),
         )
-    if len(chains) > 1:
+    if config.get("relayer"):
```

Comment:
this change maybe the closest to the relayer issue in integration tests, can try to revert it and try again.

Reply:
Ok